UML_vantuz/FIrstUMLLab/FIrstUMLLab/Passager.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FIrstUMLLab
+{
+    public class Passager : Person
+    {
+        public bool IsPayed { get; private set; } = false;
+        public Transport UseTransport { get; private set; }
+
+        public Passager(Transport transport)
```

Comment:
passenger will not always be in the transport, so remove that param and add base call at least